content/_data/solutions/android.yml

```
@@ -4,14 +4,17 @@ blog_tags:
   - android
 articles:
 -
-  - 'http://blog.zuehlke.com/en/configure-your-android-project-on-jenkins/'
-  - 'Configure your Android project on Jenkins'
+  - 'https://medium.com/simform-engineering/getting-started-with-jenkins-android-1138ee3d1e99'
+  - 'Getting Started with Jenkins + Android'
 -
   - 'https://www.bignerdranch.com/blog/continuous-delivery-for-android/'
   - 'Continuous Delivery for Android'
 -
   - 'https://dzone.com/articles/automating-continuous'
   - 'Automating Android Projects With Gradle using Jenkins on Windows'
+-
+  - 'https://www.cloudbees.com/blog/building-android-gradle-apps-jenkins'
+  - 'Building Android Gradle Apps with Jenkins'
```

```suggestion
  - 'https://www.cloudbees.com/blog/building-android-gradle-apps-jenkins'
  - 'Building Android Gradle Apps with Jenkins'
```

I don't know if this would be worth adding only because it appears to be much older and has little information in the way of android. It's more about Gradle than anything else, and even then, it doesn't really go into much detail or information beyond the extremely basic initial steps.

---

got it let me remove it

---

done !